pkg/local-storage/member/node/qos/cgroups.go

```diff
@@ -95,10 +116,18 @@ func getDeviceNumber(devicePath string) (uint64, uint64, error) {
 	return maj, min, nil
 }
 
-func writeFile(exec exechelper.Executor, filename, value string) error {
-	result := exec.RunCommand(exechelper.ExecParams{
-		CmdName: "sh",
-		CmdArgs: []string{"-c", fmt.Sprintf("echo %s >> %s", value, filename)},
-	})
-	return result.Error
+func writeFile(filename, value string) error {
```

Comment:
I think the root cause is the `writeFile` func. In this pr, the func write content into container cgroup, not OS cgroup.

Reply:
the `12519` is process in a privileged container. the file path of blkio on host is `/system.slice/containerd.service/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice`, and it does't equal to `/sys/fs/cgroup/blkio` 

```
[root@master 12519]# pwd
/proc/12519
[root@master 12519]# cat cgroup
11:cpuacct,cpu:/system.slice/containerd.service/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
10:devices:/system.slice/containerd.service/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
9:hugetlb:/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
8:blkio:/system.slice/containerd.service/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
7:memory:/system.slice/containerd.service/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
6:cpuset:/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
5:pids:/system.slice/containerd.service/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
4:net_prio,net_cls:/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
3:freezer:/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
2:perf_event:/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
1:name=systemd:/system.slice/containerd.service/kubepods-besteffort-podd019a581_0a38_4938_81d8_2dacae511559.slice:cri-containerd:be546da0994003735b936cea90735d147b7457a6b6e7cc534cf547977f03f21c
```

Reply:
Hi @lucaDev, There're linux distributions supported by Hwameistor now. I'd like Hwameistor support more others linux in the future. Before we support it, let this pr focus on IO throttling feature. To support other distributions like Talos, it maybe require some other changes and tests. It's a better way that we can do it in another pr. What do you think?

Reply:
Yeah. You're probably right. We should still try to find a good way to make it compatible soon. I have to change my test setup then :D